charts/sftd/values.yaml

```diff
@@ -0,0 +1,71 @@
+# Default values for sftd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: quay.io/wire/sftd
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# The time to wait after terminating an sft node before shutting it down. No
+# new calls will be initiated whilst a pod is terminated. This value will be
+# more useful once we support running multiple sfts behind a load-balancer
+terminationGracePeriodSeconds: 10
+
+podAnnotations: {}
+
+# This must be set explicitly by the end-user. SFTD might be behind 1-1 NAT; which is common
+# in on-prem and also on GCP; so we need to explicitly tell sftd what its public IP is that
+# it can hand out to clients to open WebRTC streams
+# mediaAddress: <external IP>
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  runAsUser: 1000
```

Comment:
Note that we're not vulnerable to https://medium.com/nttlabs/dont-use-host-network-namespace-f548aeeef575 due to this `runAsUser` setting. Though it is important that the UID doesn't conflict with an existing UID on the host.  I don't know how go guarantee that.  In systemd I would just let the daemon assign one through `DynamicUser=` but kubernetes doesn't have a similar feature :(

Reply:
Maybe we should implement the OCI spec for systemd and wrap every container runtime invocation in a service unit.